### DISCUSSION TITLE
python support?

### DISCUSSION THREAD

**Discussion by RaiAmanRai** [FrostCo/AdvancedProfanityFilter#458] [Q&A]
Hi, is there a support to run this project using python?

Thanks & cheers!

**Comment by richardfrost** [comment]
> Hi, is there a support to run this project using python?
> 
> Thanks & cheers!

Hi @RaiAmanRai, the project is a javascript, so it's not natively supported. I haven't used it before, but maybe you could try this? https://github.com/PiotrDabkowski/Js2Py
